sentences/fr/_common.yaml

```
@@ -75,19 +75,22 @@ lists:
   cover_classes:
     values:
       - in: "store[s]"
-        out: "awning"
-      - in: "store[s]"
+        out:
+          - "awning"
+          - "blind"
+          - "shade"
+      - in: "store[s] vénitien[s]"
         out: "blind"
+      - in: "(banne[s])|(auvent[s])|(store[s] banne[s])"
+        out: "awning"
       - in: "rideau[x]"
         out: "curtain"
       - in: "porte[s]"
         out: "door"
       - in: "portes[s] de garage[s]"
```

```suggestion
      - in: "portes[s] [de|du] garage[s]"
```
Is it possible to add "La porte garage" and "La porte du garage" ?

---

Just remember that the best and simplest way to be able to say "Ouvre la porte du garage" is still to name your entity "Porte du garage" or at least to give it an alias but going to the entity option